### PR TITLE
[NR-180970] Android release notes v7.3.0 and few fixes

### PR DESCRIPTION
**Pull request by ywang-nr**
* What problems does this PR solve? - Android v7.3.0 release notes and few fixes

### CODE REVIEW

**Comment by ywang-nr** [comment]
@cthomas-newrelic for engineering review. Thanks.

@Doc Team, I will mark this PR to ready for review once engineer team review is done. Thanks.

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-730.mdx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+downloadLink: 'https://download.newrelic.com/android_agent/ant/NewRelic_Android_Agent_7.3.0.zip'
+---
+
+## New
+
+* Added the ability to persist events when application is force killed.
+* Added the ability to store harvest data that previously would be lost if the application doesn't have internet connection. These harvest are then sent after an internet connection is established and the next harvest successfully uploads. This feature is disabled by default, see more details [here](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings/#android-offline-storage) to enable it. 
+* Added `setMaxOfflineStorageSize` API to set the maximum amount of data that can be stored locally. 
+* Added support for real engine platform
+* Added support for Android dynamic feature
```
Comment (ywang-nr) [inline src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-730.mdx:14]: include dynamic feature doc here, add period

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-730.mdx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+## Fix
+
+* Fixed an issue that target activity is not captured during app launch time
```
Comment (ywang-nr) [inline src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-730.mdx:18]: elaborate more here, add period

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-730.mdx (hunk 1 above)
Comment (ywang-nr) [inline src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-730.mdx:14]: Wait for Nisarg to confirm the name, add period

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-730.mdx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+releaseDate: '2024-03-04'
+version: 7.3.0
+downloadLink: 'https://download.newrelic.com/android_agent/ant/NewRelic_Android_Agent_7.3.0.zip'
+---
+
+## New
+
+* Added the ability to persist events when application is force killed.
+* Added the ability to store harvest data that previously would be lost if the application doesn't have internet connection. These harvest are then sent after an internet connection is established and the next harvest successfully uploads. This feature is disabled by default, see more details [here](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings/#android-offline-storage) to enable it. 
+* Added `setMaxOfflineStorageSize` API to set the maximum amount of data that can be stored locally.
```
Comment (ywang-nr) [inline src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-730.mdx:12]: add a link here

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-730.mdx [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+version: 7.3.0
+downloadLink: 'https://download.newrelic.com/android_agent/ant/NewRelic_Android_Agent_7.3.0.zip'
+---
+
+## New
+
+* Added the ability to persist events when application is force killed.
+* Added the ability to store harvest data that previously would be lost if the application doesn't have internet connection. These harvest are then sent after an internet connection is established and the next harvest successfully uploads. This feature is disabled by default, see more details [here](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings/#android-offline-storage) to enable it. 
+* Added `setMaxOfflineStorageSize` API to set the maximum amount of data that can be stored locally. 
+* Added support for real engine platform
```
Comment (ywang-nr) [inline src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-730.mdx:13]: Unreal platform

**Review by bradleycamacho — APPROVED** [review]

**Pull request merged**